android_app/app/src/main/res/layout/activity_main.xml

```diff
@@ -55,13 +55,22 @@
         android:progressDrawable="@drawable/seekbar_progress"
         android:thumb="@drawable/seekbar_thumb" />
 
+    <ToggleButton
+        android:id="@+id/steeringToggle"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_alignParentBottom="true"
+        android:layout_centerHorizontal="true"
+        android:layout_marginBottom="52dp"
+        android:textOff="Manual"
+        android:textOn="Automatic" />
+
     <Button
         android:id="@+id/btn1"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:layout_alignParentBottom="true"
-        android:layout_centerHorizontal="true"
-        android:layout_marginBottom="28dp"
+        android:layout_marginBottom="180dp"
```

Comment:
Try to avoid large hard coded numbers in the design xml since it won't scale well with different screen sizes.
Smaller numbers are usually okey, but 180dp may make the app very messy or even not functional. 

There are 2 ways to go about this that I know of. 

Either you make a bunch of LinearLayouts and put the other elements as children of those. LinearLayouts are pretty easy to work with and can easily be made dynamic to suit all screen sizes.

The other way is to use "dimens" to get different numbers depending on the screen size of the current device.
Explained here: https://stackoverflow.com/questions/28674267/android-app-layout-looks-bad-with-hardcoded-values

Reply:
I won't hard code the layout in the future! Thanks for the tips!